modules/core/src/main/java/org/locationtech/jts/index/strtree/EnvelopeDistance.java

```diff
@@ -1,14 +1,25 @@
+/*
+ * Copyright (c) 2019 Martin Davis.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
 package org.locationtech.jts.index.strtree;
 
 import org.locationtech.jts.geom.Envelope;
 
 /**
- * Utility functions for working with {@link Envelope}s.
+ * Functions for computing distances between {@link Envelope}s.
  * 
  * @author mdavis
  *
  */
-public class EnvelopeUtil 
+public class EnvelopeDistance 
```

Comment:
@dr-jts Is this an API change?  E.g., do we expect folks to have used EnvelopeUtil in their code?

Reply:
No., that class was only introduced in #391 yesterday.